src/pages/patientView/PatientViewPage.tsx

```diff
@@ -39,6 +39,11 @@ interface IPatientViewState {
     cnaSegmentData: any;
     mutationData: any;
     activeTabKey: Number;
+    sampleOrder?:string[];
+    sampleColors?:{ [s:string]: string};
+    sampleLabels?:{ [s:string]: string};
+    sampleTumorType?:{ [s:string]: string};
+    sampleCancerType?:{ [s:string]: string};
```

Comment:
We may need to change these properties depending on how/where we get the sample data? Aaron has been working on a class like SampleManager/SampleFactory? @alisman can you comment on this?